_source/logzio_collections/_log-sources/python.md

```diff
@@ -70,7 +69,46 @@ level=INFO
 format={"additional_field": "value"}
 ```
 
-###### Parameters
+
+##### Dict Config
```

Comment:
The dict config section appears in the docs as step 3, but it should be as part of the configuration part (step 2), as another configuration option.